### DISCUSSION TITLE
Wooley SA-028/SA-029 can t connect

### DISCUSSION THREAD

**Discussion by grizzly1105** [Koenkk/zigbee2mqtt#19270] [Devices]
Hi there. the device isn t connecting. Do you have any ideas?

**Comment by electric-m** [comment]
Same here. device is in pairing mode, but zigbee2mqtt does not a show any sign of connecting, neither in the Web IF, nor in the console of the docker container. Miami temp sensor works just fine, so i recon it's not the problem of zigbee2mqtt?
